deeplake/api/dataset.py

```diff
@@ -1386,14 +1386,19 @@ def ingest_classification(
             if os.path.isdir(dest) and os.path.samefile(src, dest):
                 raise SamePathException(src)
 
-            if src.endswith(".csv"):
+            if src.endswith(".csv") or src.endswith(".txt"):
```

Comment:
```suggestion
            if src.lower().endswith(".csv") or src.endswith(".txt"):
```

Reply:
@farizrahman4u os there a reason `.lower()` is not added to the `.txt` part?